### PR TITLE
Fix dashboard breakage

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

See https://github.com/elastic/integrations/issues/505

I'm still trying to test this, as I don't have a great windows environment set up, but if anyone does have an at-hand winlogbeat test environment, please pull this down and test.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [ ] I have verified that all datasets collect metrics or logs.

## How to test this PR locally

-  Pull down PR
- Build the system package with `elastic-package build`
- Start the stack, with an agent on Windows
- Make sure the Windows dashboards is working.

### CODE REVIEW

**Comment by mtojek** [comment]
I think there is a problem with formatting?

```
[2021-01-13T19:39:46.878Z] + ../../build/elastic-package check -v
[2021-01-13T19:39:46.878Z] elastic-package has been installed.
[2021-01-13T19:39:46.878Z] 2021/01/13 19:39:46 DEBUG Enable verbose logging
[2021-01-13T19:39:46.878Z] Format the package
[2021-01-13T19:39:46.878Z] Error: checking package failed: formatting the integration failed (path: /var/lib/jenkins/workspace/gest-manager_integrations_PR-522/src/github.com/elastic/integrations/packages/system, failFast: true): walking through the integration files failed: formatting file failed (path: /var/lib/jenkins/workspace/gest-manager_integrations_PR-522/src/github.com/elastic/integrations/packages/system/data_stream/application/fields/ecs.yml): file is not formatted (path: /var/lib/jenkins/workspace/gest-manager_integrations_PR-522/src/github.com/elastic/integrations/packages/system/data_stream/application/fields/ecs.yml)
script returned exit code 1
```

File: packages/system/kibana/visualization/system-Event-Levels.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
                 "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.index",
                 "query": {
                     "language": "kuery",
-                    "query": "(data_stream.dataset:windows.application OR data_stream.dataset:windows.forwarded OR data_stream.dataset:windows.powershell OR data_stream.dataset:windows.powershell_operational OR data_stream.dataset:windows.security OR data_stream.dataset:windows.sysmon_operational OR data_stream.dataset:windows.system)"
+                    "query": "(data_stream.dataset:windows.application OR data_stream.dataset:windows.forwarded OR data_stream.dataset:windows.powershell OR data_stream.dataset:windows.powershell_operational OR data_stream.dataset:windows.security OR data_stream.dataset:windows.sysmon_operational OR data_stream.dataset:windows.system OR data_stream.dataset:system.application OR data_stream.dataset:system.security OR data_stream.dataset:system.system)"
```
Comment (narph) [inline packages/system/kibana/visualization/system-Event-Levels.json:10]: In they future we might want to remove the windows `powershell` filters and try to add any meaningful visualizations back in the windows module. We can approach this in the migration of the windows package

**Review by narph — APPROVED** [review]
I have not tested this but looks like the filters for the system events have been added. LGTM

File: packages/system/kibana/visualization/system-Event-Levels.json (hunk 1 above)
Comment (fearful-symmetry) [reply packages/system/kibana/visualization/system-Event-Levels.json:10]: Yah, this doesn't seem like a great long-term solution, but I'm a tad worried about breaking visualizations for pre-7.11 data.

**Comment by fearful-symmetry** [comment]
Okay, so, done some testing and things appear to be working. My biggest concern is that my limited Windows system knowledge means that I can't quite tell the difference between fields that are unpopulated because there's no data and fields that are unpopulated because something might be broken.

**Pull request merged**